### DISCUSSION TITLE
server config

### DISCUSSION THREAD

**Discussion by robvanmieghem** [threefoldtech/web3_proxy#59] [Ideas]
Currently settings like the Ethereum node the server should interact with is passed from the client.
While this is not only cluttering the client api, the client does not need to know and often can not even know such settings.

Wouldn't it be better to configure the server ( through a config file or parameters) for things like this?

**Comment by DylanVerstraete** [comment]
An Ethereum node can be deployed using the web3proxy, so we don't want all of this config beforehand.
Also a client is loaded with a url and this connection state is saved on the server
